cmd/kubeadm/app/phases/apiconfig/clusterroles.go

```diff
@@ -19,10 +19,11 @@ package apiconfig
 import (
 	"fmt"
 
+	"k8s.io/api/core/v1"
+	rbac "k8s.io/api/rbac/v1beta1"
+rbachelper "k8s.io/kubernetes/pkg/apis/rbac/v1beta1"
```

Comment:
Is that GitHub displaying the tabs incorrectly, or is "rbachelper" really at the start of the line?